src/setup.ts

```diff
@@ -14,7 +14,7 @@ function urlExt(s: string): string {
   const components = u.path?.split('/')
   if (components && components?.length > 0) {
     const last = components[components?.length - 1]
-    const dot = last.indexOf('.')
+    const dot = last.lastIndexOf('.')
```

Comment:
As the version format contains dots (`.`) `urlExt()` was returning `.3-2021.07.tar.bz2` as the extension. So with this change we only look at the last part of the extension.

This also required a change in the switch case below to match `.bz2` instead of `.tar.bz2`.